packages/runtime-common/helpers/strip-scoped-css-glimmer-attributes.ts

```diff
@@ -0,0 +1,7 @@
+export default function stripScopedCSSGlimmerAttributes(compiledTemplateString: string) {
+  let attributeArray = `\\[(14|24),\\\\"data\\-scopedcss\\-[0-9a-f]{10}\\\\",\\\\"\\\\"\\]`;
+  let double = new RegExp(`\\[${attributeArray}\\]`, 'g');
+  let single = new RegExp(`${attributeArray},`, 'g');
+
+  return compiledTemplateString.replace(double, 'null').replace(single, '');
+}
```

Comment:
While I don’t love storing Glimmer opcodes here to strip out the `data-scopedcss-*` attributes here I find it preferable to updating the assertions to expect the attributes. I’m open to removing these helpers if people think the data attributes should be asserted on.